src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -23,7 +24,8 @@ public record CommandContext<T extends SchemaObject>(
     EmbeddingProvider embeddingProvider,
     String commandName,
     JsonProcessingMetricsReporter jsonProcessingMetricsReporter,
-    ApiFeatures apiFeatures) {
+    ApiFeatures apiFeatures,
+    OperationsConfig operationsConfig) {
```

Comment:
OperationsConfig contains limits needed by Shredder for validation (or, eventually separate validation component); not yet available efficiently from CommandContext before changes.